File: two_sum.py
def add_Sum(arr, target):
    for i in range(len(arr)):
        for j in range(i + 1, len(arr)):
            if arr[i] + arr[j] == target:
                return [i, j]
    print("No solution found")
    return []

nums = [2, 7, 11, 15]
target = 9
print(add_Sum(nums, target))  

File: two_sum.js
const addSum = (nums, target) => {
  for (i = 0; i < nums.length; i++)
    for (j = i + 1; j < nums.length; j++) {
      if (nums[i] + nums[j] === target) {
        return [i, j];
      }
    }
  console.log("No solution found");
  return [];
};

const nums = [2, 7, 11, 15], target = 9;

console.log(addSum(nums, target));
